File: my-journey/dsa/week-01/python/L-the-brothers.py
# Problem: L. The Brothers
# Pattern:
# Link: https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/L
# Difficulty:
# Time:  O()
# Space: O()

import sys
input = sys.stdin.readline


def solve():
    first1, last1 = input().split()
    first2, last2 = input().split()
    # your logic here
    if last1 == last2:
        print("ARE Brothers")
    else:
        print("NOT")


if __name__ == "__main__":
    solve()

File: my-journey/dsa/cp/week-01/cpp/L-the-brothers.cpp
// Problem: L. The Brothers
// Pattern:
// Link: https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/L
// Difficulty:
// Time:  O()
// Space: O()

#include <iostream>
using namespace std;

void solve() {
    string first1, last1, first2, last2;
    cin >> first1 >> last1 >> first2 >> last2;
    // your logic here
    if (last1 == last2)
        cout << "ARE Brothers\n";
    else
        cout << "NOT\n";
}

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    solve();
    return 0;
}
